### PR TITLE
Update devices.js - ROB_200-010-0

### PR DESCRIPTION
**Pull request by Chamondgl**
Small update removing an unneeded genPowerCfg bit and inverting cover status state reporting.

### CODE REVIEW

**Comment by Chamondgl** [comment]
I have an issue with action buttons being greyed out as well. https://github.com/Koenkk/zigbee-herdsman-converters/issues/1467

File: devices.js [hunk 1]
```diff
@@ -5553,12 +5553,12 @@ const devices = [
         vendor: 'ROBB',
         description: 'Zigbee curtain motor controller',
         supports: 'open, close, stop, position',
-        meta: {configureKey: 1},
+        meta: {configureKey: 2},
```
Comment (Koenkk) [inline devices.js:5556]: To fix https://github.com/Koenkk/zigbee-herdsman-converters/issues/1467 try changing this line to

```
meta: {configureKey: 2, coverInverted: true},
````

Comment (Chamondgl) [reply devices.js:5556]: Thanks. That doesn't seem to do anything. Even after a re-pair it doesn't invert the reported state.

Comment (Chamondgl) [reply devices.js:5556]: It seems to magically work now. What an adventure.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**